model/textparse/openmetricsparse_test.go

```diff
@@ -64,7 +64,24 @@ _metric_starting_with_underscore 1
 testmetric{_label_starting_with_underscore="foo"} 1
 testmetric{label="\"bar\""} 1
 # TYPE foo counter
-foo_total 17.0 1520879607.789 # {id="counter-test"} 5`
+foo_total 17.0 1520879607.789 # {id="counter-test"} 5
+foo_created 1000
+foo_total{a="b"} 17.0 1520879607.789 # {id="counter-test"} 5
+foo_created{a="b"} 1000
+# TYPE bar summary
```

Comment:
This is one of the cases which hasn't been covered yet by the `CreatedTimestamp` method as the deepcopy starts from where the parser is

eg: if have this case, how do we make the deep copy move backwards? (ig we just go in reverse until we hit TYPE text or reach another metrics)
```
# TYPE bar summary
bar_count 17.0
bar_created 1520430000        
bar{quantile="0.95"} 123.7     <---- Parser and deepCopy start from here
bar{quantile="0.99"} 150.0     <---- 
bar_sum 324789.3               <---- deepCopy loops till here and stops
```

Reply:
Hm, but we are sure deepCopy (so CreatedTimestamp) is invoked on every EntrySeries, no?

Reply:
so yea, definitely let's add those cases here, even if we fail for now (: If we decide to fail because of reasons, it's ok to have tests that ... assert failure, so everyone knows what we  don't (yet) support. (: